api/search_test.go

```diff
@@ -24,13 +25,22 @@ var (
 	topic               = "testing"
 )
 
+// TODO this should really be mocked
+func getTestProducer() kafka.Producer {
```

Comment:
The Kafka producer will need to be mocked for tests to pass on the build server.